src/Microsoft.ML.TimeSeries/SrCnnAnomalyDetectionBase.cs

```diff
@@ -292,6 +358,282 @@ private Single CalculateSocre(Single mag, Single avgMag)
                     }
                     return (float)(Math.Abs(mag - avgMag) / safeDivisor);
                 }
+
+                private List<int> GetAnomalyIndex(List<Single> magList, List<Single> avgMagList)
+                {
+                    List<int> anomalyIdxList = new List<int>();
+                    for (var i = 0; i < magList.Count; ++i)
+                    {
+                        double safeDivisor = avgMagList[i];
+                        if (safeDivisor < 1e-8)
+                        {
+                            safeDivisor = 1e-8;
+                        }
+                        var score = (Math.Abs((double)magList[i] - (double)avgMagList[i]) / safeDivisor);
+                        if (score > 3.5f)
```

Comment:
>3.5f [](start = 36, length = 4)

What is the significance of this value? Can you add a comment, or define this as a constant somewhere?